downloadDirectory/src/main/java/org/reactome/release/downloadDirectory/CreateReleaseTarball.java

```diff
@@ -0,0 +1,87 @@
+package org.reactome.release.downloadDirectory;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Paths;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.eclipse.jgit.api.Git;
+import org.eclipse.jgit.api.errors.GitAPIException;
+
+public class CreateReleaseTarball {
+	private static final Logger logger = LogManager.getLogger();
+	
+	public static void execute(int releaseNumber) throws IOException, InterruptedException {
+		logger.info("Running CreateReleaseTarball");
+		String tarDirRelease = "reactome_tar/" + releaseNumber + "/Release";
+		String absReleaseDir = "/usr/local/gkb/scripts/release/website_files_update";
```

Comment:
The values for `absReleaseDir` and `absReactomeDir` should be configurable, set in a properties file.